build/distributions/CentOS/7/x86_64/yum_data/packages.yaml

```diff
@@ -130,22 +121,19 @@ puppetserver:
   :source: https://yum.puppetlabs.com/el/7/PC1/x86_64/puppetserver-2.8.1-1.el7.noarch.rpm
 python-linecache2:
   :rpm_name: python-linecache2-1.0.0-1.el7.noarch.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/p/python-linecache2-1.0.0-1.el7.noarch.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/p/python-linecache2-1.0.0-1.el7.noarch.rpm
 python-redis:
   :rpm_name: python-redis-2.10.3-1.el7.noarch.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/p/python-redis-2.10.3-1.el7.noarch.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/p/python-redis-2.10.3-1.el7.noarch.rpm
 python-simplejson:
   :rpm_name: python-simplejson-3.3.3-1.el7.x86_64.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/p/python-simplejson-3.3.3-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/p/python-simplejson-3.3.3-1.el7.x86_64.rpm
```

Comment:
I didn't see python-simplejson at that URL, but did see it in packagecloud in the 6_X_Dependencies repo